_dashboards/management/management-index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Dashboards Management
+nav_order: 110
+has_children: true
+---
+
+# Dashboards Management
+Introduced 2.10
+{: .label .label-purple }
+
+Dashboards Management serves as the command center for customizing OpenSearch Dashboards to your needs. A view of the interface is shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboards-management-ui.png" alt="Dashboards Management interface" width="700"/>
+
+{::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} **Note**<br>OpenSearch and OpenSearch Dashboards privileges govern access to individual features. If you do not have the appropriate access, consult your administrator. 
+{: .note}
+
+## Applications
+
+The following applications are available in Dashboards Management:
+
+- **Index Patterns:** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. An index pattern can point to one or more indexes, data streams, or index aliases. 
+- **[Data Sources]({{site.url}}{{site.baseurl}}/dashboards/management/multi-data-sources/):** You can use Data Sources to specify the source configuration in your copy of the [OpenSearch Dashboards configuration file]({{site.url}}{{site.baseurl}}https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml).
+- **Saved Objects:** The Saved Objects interface helps you organize and manage your saved objects. Saved objects are files that store data for later use, such as dashboards, visualizations, and maps.
+- **[Advanced Settings]({{site.url}}{{site.baseurl}}/dashboards/management/advanced-settings/):** The Advanced Settings interface gives you the flexibility to personalize the behavior of OpenSearch Dashboards.
```

Comment:
I would expand a little here on what the difference is between data sources and advanced settings. Otherwise, to a first-time user, the descriptions might sound similar.